src/materials/shaders/pointcloud.frag

```diff
@@ -61,6 +65,20 @@ void main() {
 	vec3 color = vColor;
 	float depth = gl_FragCoord.z;
 
+	#if defined (clip_horizontally) || defined (clip_vertically)
+		vec2 ndc = vec2((gl_FragCoord.x / screenWidth), (gl_FragCoord.y / screenHeight));
+
+		if(!(ndc.x > clipExtent.x && ndc.x < clipExtent.y))
+		{
+			discard;	
```

Comment:
We already have a way of discarding points ( maybe take a look at the normal filtering part of the code).
 
If I am not mistaken, if we discard points like this, our point picking will still see these invisible points.

Reply:
Good point. I'll check if that would potentially cause problems for our use case. Thx!